austation/code/modules/mob/living/simple_animal/friendly/drone/_drone.dm

```diff
@@ -27,4 +27,22 @@
 	if(istype(J))
 		return J
 
+/obj/item/drone_shell
+	name = "hand-held drone shell"
+	desc = "A drone shell yet to be activated. Use it in your hand to activate the drone shell."
+	w_class = WEIGHT_CLASS_NORMAL
+	icon = 'icons/mob/drone.dmi'
+	icon_state = "drone_maint_grey"
+
+/obj/item/drone_shell/attack_self(mob/user)
+	. = ..()
+	var/turf/T = get_turf(src)
+	if(!T)
+		user.show_message("<span class='warning'> There is no room to build \the [src].</span>")
+		return
 
+	user.visible_message("<span class='notice'> [user] builds \a [src].</span>", "<span class='notice'> You build \the [src].</span>", "<span class='notice'> You can hear someone building \a [src].</span>", 7)
```

Comment:
```suggestion
	user.visible_message("<span class='notice'>[user] activates \a [src].</span>", "<span class='notice'>You activate \the [src].</span>")
```
removes some errant spaces that will make the chat output look gross, removes the audible message and fixes wording

Reply:
You don't like the audible part? Ok.